opencti-platform/opencti-front/src/private/components/analyses/reports/ReportDetails.jsx

```diff
@@ -168,7 +168,8 @@ const ReportDetailsComponent = (props) => {
           {t('Correlated reports')}
         </Typography>
         <List>
-          {R.take(expanded ? 200 : 5, report.relatedContainers.edges)
+          {(report.relatedContainers?.edges ?? []).length >= 2
```

Comment:
I think you can write something like this instead :
data.relatedContainers?.edges?.length >= 2